src/SaveObject.ts

```diff
@@ -369,7 +436,7 @@ function evaluateVersionCompatibility(ver: string | number): void {
     Player.reapplyAllSourceFiles();
   }
   if (ver < 3) {
-    anyPlayer.money = parseFloat(anyPlayer.money);
+    anyPlayer.money = parseFloat(anyPlayer.money as string);
```

Comment:
What lint error triggers here? `parseFloat` should be fine to run on any type.

Reply:
`parseFloat` coerces the parameter to a string, so technically it's fine to pass anything and rely on JS's type coercion. However, in TypeScript, `parseFloat` explicitly requires a string.